design_docs/Hail_Filter_and_Annotation.md

```diff
@@ -0,0 +1,67 @@
+# Hail Query Stage
+
+This stage of the workflow uses hail query to load, annotate, and apply provisional categories to variants.
+
+[hail_filter_and_classify.py](../reanalysis/hail_filter_and_classify.py)
+
+## Stage Details
+
+Runs within a Hail Batch, as a Hail Query runtime. For now, due to functional constraints, the process runs inside
+DataProc on GCP. We are aiming to split the HailQuery/VEP annotation stage from this infrastructure requirement soon.
+
+This stage takes a Hail MatrixTable representation of a VCF as input, applies VEP annotation, and adds provisional
+category tags. Flow chart representations of the logic for pre-and post annotation filters are shown below
+
+![Pre-Annotation Filters](images/hail_pre_filter.png)
```

Comment:
This image looks like it's been upscaled from a really low-res version?

Reply:
These were terrible screenshots from Miro that I had to resize. I've added some better ones now